bin/lib/zopen-build

```diff
@@ -989,6 +997,12 @@ resolveCommands()
   if [ "${ZOPEN_INSTALL_DIR}x" = "x" ]; then
     export ZOPEN_INSTALL_DIR="${HOME}/zopen/prod/${ZOPEN_NAME}"
   fi
+  if [ "${ZOPEN_LOG_DIR}x" = "x" ]; then
```

Comment:
I see some places we use -z/-n and others we use this old form of null string checking.   Since this is a bash script,  shall we use -z/-n?  I find it more readable and should be faster.
```suggestion
  if [[ -z "${ZOPEN_LOG_DIR}" ]]; then
```

Reply:
We can probably switch to bash once @MikeFultonDev makes the zopen-setup script available. For now, we're still using /bin/sh

Reply:
These work in /bin/sh.  See lines 837, 954, etc.  The code as is is fine.  It matches may other comparisons.